source/README.adoc

```diff
@@ -88,6 +88,8 @@ Sources are Kubernetes objects. In addition to the standard Kubernetes
 | `queue_config.delete_when_unused` {optional} | Boolean
 | `queue_config.exclusive` {optional} | Boolean
 | `queue_config.nowait` {optional} | Boolean
+| `channel_config.prefetch_count` {optional} | Int that limits the https://www.rabbitmq.com/consumer-prefetch.html[Consumer Prefetch Value].
```

Comment:
This explains the environment variable name chosen here https://github.com/knative-sandbox/eventing-rabbitmq/pull/522/files#r754456187

I am still not convinced that exposing this much RabbitMQ-specific detail is a good idea. If these broker types are meant to be easily interchangeable, then the current implementation-specific approach is going against that principle. How are you thinking about this @benmoss @salaboy ?